lib/client.js

```diff
@@ -1942,6 +1942,16 @@ function doInitialSync(client, historyLen, includeArchived) {
                     room.addReceipt(receipts[j]);
                 }
 
+                var privateUserData = data.rooms[i].private_user_data || [];
+                for (j = 0; j < privateUserData.length; j++) {
+                    var event = _PojoToMatrixEventMapper(client)(privateUserData[j]);
+                    if (event.type === "m.tag") {
```

Comment:
`event` at this point should be a `MatrixEvent` in which case we should be using `getType()` here.